recipes/libsvm/3.16/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 make
-mv svm-train $PREFIX
-mv svm-scale $PREFIX
-mv svm-predict $PREFIX
+mkdir $PREFIX/bin
```

Comment:
I think I once saw conda not creating the bin folder, so I usually add ``mkdir -p $PREFIX/bin``. But we can remove it and see if it works now.